src/scripts/main.js

```diff
@@ -353,8 +353,21 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+const table = document.querySelector('table');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((el) => {
+  el.Age = el.died - el.born;
+  el.Century = Math.ceil(el.died / 100);
+  el.sex = el.sex === 'y' ? 'female' : 'male';
```

Comment:
There is a typo in your code. You're checking if `el.sex` is equal to 'y' instead of 'f'. It should be 'f' for female.